### PR TITLE
Reorder the entries of the log for easier reading

### PR DESCRIPTION
**Pull request by nickvergessen**
Downstream 27562

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]
Fine, as long as this doesn't break any external systems like log analyzers.

**Comment by MorrisJobke** [comment]
> Fine, as long as this doesn't break any external systems like log analyzers.

They should use son parsing anyways (which is in most the systems the case and then the order doesn't matter anymore.

File: lib/private/Log/File.php [hunk 1]
```diff
@@ -106,16 +106,16 @@ public static function write($app, $message, $level) {
...
+			'remoteAddr',
+			'user',
+			'app',
 			'method',
 			'url',
-			'user',
+			'message',
 			'userAgent',
-			'version'
+			'version',
```
Comment (MorrisJobke) [inline lib/private/Log/File.php]: This breaks the `compact` - let me fix it.

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

**Pull request merged**